iina/en.lproj/PrefGeneralViewController.strings

```diff
@@ -123,7 +123,7 @@
 "jeK-46-2BG.title" = "History:";
 
 /* Class = "NSButtonCell"; title = "Show artist and track name for audio files when avaliable"; ObjectID = "kBI-Th-bBV"; */
```

Comment:
Please also correct it in the comment for consistency purposes

Reply:
Fixed